gradle/witness/gradle-witness.gradle

```diff
@@ -12,6 +12,9 @@
 
 dependencyVerification {
     verify = [
+        'bisq:core:4d0e437026e53968298710acd8bc2e4b1b0e1d37859c69359cf63e073ae02b86',
+        'bisq:p2p:c36dde32292ca7335ea594e0cf36f8f766cd3689cd3886b095def3ff946ed60b',
+        'bisq:common:d9fef12428771154ca70d9299a36feb1d855b07242822a96a16f43260af03149',
         'org.controlsfx:controlsfx:b98f1c9507c05600f80323674b33d15674926c71b0116f70085b62bdacf1e573',
```

Comment:
Indeed. https://github.com/bisq-network/bisq/blob/9389a8e1fa5005d16771116599da9b5da13ad9ce/gradle/witness/gradle-witness.gradle#L6 should be changed to exclude `bisq:` instead of `network.bisq` libraries.

So it should be change to `... | grep -v 'bisq:'`

Reply:
I use now:
`./gradlew -q calculateChecksums | grep -v 'bisq:\|org.openjfx:' >> gradle/witness/gradle-witness.gradle`

That filters only out the local bisq projectes but not bitcoinj and others as well org.openjfx libs which cause issues.